initializer.sh

```diff
@@ -118,7 +117,7 @@ cd ~/.radicle/seed/creature-pigeon
         # Peers ID
         peer_one=$RADICLE_PEER_ONE
         peer_two=$RADICLE_PEER_TWO
-        peer_three=$RADICL_PEER_THREE 
+        peer_three=$RADICL_PEER_THREE
```

Comment:
I had a typo here, `$RADICL_PEER_THREE` is missing the `E`. It should be `$RADICLE_PEER_THREE`.
Addtionally, since you have added the fourth peer in the Dockerfile, we need to add `peer_four=$RADICLE_PEER_FOUR` in order to successfully add that fourth peer into the node.

Reply:
Ok I will fix that